docs/operator-config.asciidoc

```diff
@@ -0,0 +1,61 @@
+[id="{p}-operator-config"]
+== Operator configuration
+
+The operator exposes several configuration options. Unless otherwise noted, they can be set as environment variables or command line flags.
+
+
+[width="70%",valign="middle",halign="center",options="header"]
+|==========================
+|Flag |Type|Default|Description
+
+|log-verbosity |int |0 |Verbosity level of logs. -2=Error, -1=Warn, 0=Info, >0=Debug
+|enable-debug-logs |bool |false |Enables debug logs. Equivalent to `log-verbosity=1`
```

Comment:
Unrelated to that PR: that 2nd option and the first one feel weird. Looks like the first option would be enough? I don't remember if this is something we discussed @charith-elastic?

Reply:
@david-kow raised it in the original PR but we didn't really go into detail, might be worth revisiting
https://github.com/elastic/cloud-on-k8s/pull/1699#discussion_r322726830